src/containers/Login/index.tsx

```diff
@@ -83,14 +83,10 @@ const Login = ({ toggleSnackbar, setSnackbarMessage, setLoading }: ILoginProps)
       try {
         console.log('Trying to authenticate ceramic')
         await authenticateCeramic(ceramic, composeClient)
+        navigate('/')
       } catch (e) {
         console.log(`Error trying to authenticate ceramic: ${e}`)
       }
-      if (location.state?.from) {
```

Comment:
you shouldn't remove this part ,it navigates you to the link you entered after logging